ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/mappers/ClaimMapper.java

```diff
@@ -61,6 +64,11 @@ private CountyCourtJudgment toNullableCountyCourtJudgment(String input) {
         return toNullableEntity(input, CountyCourtJudgment.class);
     }
 
+    private List<String> toNullableFeatures(String input) {
```

Comment:
Nothing indicates this is for features. Name could be made more generic e.g. `toList`

Reply:
done